frontend/common/.storybook/decorators/MockGraphqlDecorator.tsx

```diff
@@ -23,6 +24,8 @@ export default function MockGraphqlDecorator(Story: StoryFn, context: StoryConte
   // Source: https://johnclarke73.medium.com/mocking-react-context-in-storybook-bb57304f2f6c
   // See: https://storybook.js.org/docs/react/addons/addons-api#useparameter
   const responseData: any = useParameter('apiResponses', {})
+  const defaultConfig = { minTimeout: 0, maxTimeout: 0 };
```

Comment:
This confused me a bit when I first saw it.  Could you add a comment?  Something like "By default, no simulated network delay"